udp_mux.go

```diff
@@ -365,13 +365,12 @@ func (m *UDPMuxDefault) connWorker() { //nolint:gocognit
 		var destinationConn *udpMuxedConn
 		m.addressMapMu.Lock()
 		if conns, ok := m.addressMap[addr.String()]; ok {
-			if localHost.IsUnspecified() {
+			destinationConn, ok = conns[ipAddr(localHost.String())]
```

Comment:
Does this break Windows build again @cnderrauber ? You had made this change to fix Windows build in https://github.com/pion/ice/commit/12148c5d62514bcfc88b2c47a6a7a56a7774e3a3. Would this be an issue again now?

Reply:
No, windows should work well for this